scripts/JSRootPainter.more.js

```diff
@@ -3158,8 +3158,11 @@
          histo.fTitle = mgraph.fTitle;
          histo.fXaxis.fXmin = uxmin;
          histo.fXaxis.fXmax = uxmax;
-      }
 
+         //I think it's probably a good idea to save it? Real ROOT does something like this, I think. 
+         mgraph.fHistogram = histo; 
+     }
+ 
       histo.fYaxis.fXmin = minimum;
```

Comment:
@cozzyd 
Do you really need this?
I prefer not to set fHistogram - it makes more difficult to update object from the server

Reply:
Is there another way of accessing the temporary histogram in the post-draw callback? I couldn't figure out how to, but that may be my fault. Otherwise, I don't know how to set the axis titles or time display on a TMultiGraph created in JavaScript.

Reply:
> Is there another way of accessing the temporary histogram in the post-draw callback?

Look here https://github.com/root-project/jsroot/blob/dev/scripts/JSRootPainter.more.js#L3236
There is "firstpainter" object which is painter of histogram. Just use `firstpainter.GetHisto()`

Reply:
Thanks, I added a commit reverting the saving to fHistogram